config.go

```diff
@@ -21,6 +21,7 @@ type Config struct {
 	MetricMaxLength     int       `yaml:"metric_max_length"`
 	NumReaders          int       `yaml:"num_readers"`
 	NumWorkers          int       `yaml:"num_workers"`
+	OmitEmptyHostname   bool      `yaml:"omit_empty_hostname"`
```

Comment:
Could you add this to the `README.md` as well? An entry in the `Configuration` section would be great. You could also write a para about it if you think it's worth calling out.